pkg/controller/mcmhub/hook_test.go

```diff
@@ -262,14 +261,14 @@ var _ = Describe("given a subscription pointing to a git path without hook folde
 			u := &subv1.Subscription{}
 
 			if err := k8sClt.Get(ctx, subKey, u); err != nil {
-				return fmt.Errorf("failed to get subscription %s, err: %s", subKey, err.Error())
+				return fmt.Errorf("failed to get subscription %s, err: %w", subKey, err)
 			}
 
 			fmt.Printf("subscription= %+v\n", u)
 
 			appsubReport := &appsubReportV1alpha1.SubscriptionReport{}
 			if err := k8sClt.Get(ctx, appsubReportKey, appsubReport); err != nil {
-				return fmt.Errorf("failed to get the appsub AppsubReport %s, err: %s", appsubReportKey, err.Error())
+				return fmt.Errorf("failed to get the appsub AppsubReport %s, err: %w", appsubReportKey, err)
```

Comment:
Why is this change needed? It seems fine before?

Reply:
This change came from adding the linter, `errorlint` which recommended a non-wrapping format as well when researching why it wanted %w I came along this: https://stackoverflow.com/a/61287626